core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/hashJoin/HashJoinOpExec.scala

```diff
@@ -28,6 +28,30 @@ class HashJoinOpExec[K](
   var currentEntry: Iterator[Tuple] = _
   var currentTuple: Tuple = _
 
+  def getBuildHashTable(): ArrayBuffer[mutable.HashMap[K, ArrayBuffer[Tuple]]] = {
```

Comment:
the method name is confusing. by reading the code I think it is "selecting map entries and build a new map to be sent to another worker". if my understanding is correct, it does not match the name `getBuildHashTable`, at least not clear. can we rename it to something else to make it clear?

Reply:
renamed it to `getBuildHashTableBatches`